src/mapcss/Eval.py

```diff
@@ -15,6 +15,10 @@
 #   You should have received a copy of the GNU General Public License
 #   along with kothic.  If not, see <http://www.gnu.org/licenses/>.
 
+import logging
+
+logger = logging.getLogger('mapcss.Eval')
+logger.setLevel(logging.ERROR)
 
 class Eval():
```

Comment:
This class is responsible for `eval(...)` expressions in .mapcss files. But Organic Maps never used evals in styles.
Should we remove eval completely from the project?